### ISSUE TITLE
Unable to build the project in intellji

### ISSUE THREAD

**Issue by odatadev2021** [davidmoten/odata-client#2]
Can you please help with the steps? Unable to build the project and gives error.

**Comment by odatadev2021** [comment]
Any comments/feedback

**Comment by davidmoten** [comment]
feel free to add the advice I gave you in an email to this page.

**Comment by davidmoten** [comment]
As per my email to you 21 Feb:

I just imported the whole lot into Intellij and ran mvn clean install in the IDE. When you import the odata-client directory as Maven, all the child modules get added automatically and are visible in the Maven view in IntelliJ:

![1](https://user-images.githubusercontent.com/318187/76811485-bf50d800-6845-11ea-8c4e-81f26ee1ff34.png)

Then to do full build of whole project select all the modules on the right and click on the m button. Run 'mvn clean install'.

![2](https://user-images.githubusercontent.com/318187/76811510-e3acb480-6845-11ea-8139-8dbb9ca47fda.png)

If you are on Java 11 then you'll need to do a git pull because I've updated with support for Java 11+.
